specification/monitor/resource-manager/readme.md

```diff
@@ -36,10 +36,9 @@ title: MonitorManagementClient
 description: Monitor Management Client
 openapi-type: arm
 tag: package-2022-06
-
```

Comment:
> > Hi @avjerafi , for the "Swagger LintDiff" and "SDK Track2 Validation" Error, both of them are complaining about incompatible Schema. They are about defining/referencing multiple models which has same name, and they are incompatible, Would you please check?
> 
> Hi @changlong-liu, Yes, I can see the cause: there are duplicate type definitions which are not identical. But these involve several resource types, which are not handled by my team and I cannot modify. The error messages include a suggestion to use the "modelerfour.lenient-model-deduplication" setting to temporarily suppress these comparisons. How do I set this? I could not find this setting being used anywhere in the swagger spec.

You may try to adding lenient here as:
~~~
modelerfour:
  lenient-model-deduplication: true
~~~

Reply:
Thank you @changlong-liu.

I've applied the flag and the "SDK Track2 Validation" as you directed, and the error was suppressed.

However, LintDiff validation error is still not suppressed, even though they both refer to the same issue, as far as I can tell. I cannot completely resolve this within the scope of this PR because it involves cooperation with many teams in order to align all the types used in all the different resource types.

This effort will certainly be part of the next SDK release, which we are working on these days in parallel to publishing this new api version.

Please advise.
Thanks!

Reply:
We should not add this flag here, this will silently take effect in every SDK, and SDKs will not want this flag.
For Go and DotNet SDK, if your swagger must have this flag to be generated correctly, you will never GA.
Therefore if the purpose of adding this flag is just to "suppress" the track 2 validation failure, please do not do that, it only suppress the failure instead of fixing it. The failure in that pipeline is something the SDK owners would like to know and deal with. Please remove this flag and let it fail